packages/react-native-reanimated/src/layoutReanimation/animationBuilder/Keyframe.ts

```diff
@@ -105,7 +105,7 @@ class InnerKeyframe implements IEntryExitAnimationBuilder {
     const duration: number = this.durationV ? this.durationV : 500;
     const animationKeyPoints: Array<number> = Array.from(
       Object.keys(this.definitions)
-    ).map(parseInt);
+    ).map((value) => parseInt(value, 10));
```

Comment:
I think this change deserves it to be made in a separate PR.

Reply:
👍